### PR TITLE
Fixes attach_limb() not calling update_disabled()

### LINKED ISSUE #56141: Digitigrades are immune to leg paralysis
## Reproduction:

Separated this issue from #56076 .

Digitigrade Lizards, are immune to having either or both their legs disabled via brain trauma. This includes from the roundstart paraplegic quirk (Free +2 points for digi lizards, wowee)

### LINKED ISSUE #59671: Wheelchair FREE using the power of the prosthetic quirk
## Round ID:
[164151](https://scrubby.melonmesa.com/round/164151)

## Testmerges:
None

## Reproduction:
Become wheelchair bound, and hope for a cool robot arm. Get robot leg and score neither.

### PR DESCRIPTION
**Pull request by Ghommie**
## About The Pull Request
In layman terms: Fixes intrinsically paralytic mobs being cured from paralysis just because they got a brand new (prosthetic) limb.
Yes, it's exactly one line, taken from an older PR i made on citadel because of roundstart slimepeople.

## Why It's Good For The Game
This will fix  #59671 and fix #56141.

## Changelog
:cl:
fix: Fixes intrinsically paralytic mobs being cured from their paralysis just because they got a brand new (prosthetic) limb.
/:cl:

### CODE REVIEW

**Comment by Mothblocks** [comment]
its 2 lines ye liar

**Comment by Ghommie** [comment]
> 
> 
> its 2 lines ye liar

Thanks for correcting me.
